test/suite/wtthread.py

```diff
@@ -56,6 +59,8 @@ def reached_max_count(self):
         return self._max_count > 0 and self.checkpoint_count >= self._max_count
 
     def run(self):
+        # Save the current test case object in thread local storage.
```

Comment:
Could you update this comment to describe why we need to perform the set() in the run() e.g. we are in a new thread context in run() thus we need set() here.